capability/test/test_hlz_tools/HLZTouchdownPointsTestCase.py

```diff
@@ -0,0 +1,305 @@
+# coding: utf-8
+'''
+-----------------------------------------------------------------------------
+Copyright 2015 Esri
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-----------------------------------------------------------------------------
+
+==================================================
+HLZTouchdownPointsTestCase.py
+--------------------------------------------------
+requirements:
```

Comment:
Any minimum requirement? ex. ArcGIS 10.X (note: for these I wouldn't call out a specific version unless it really depended upon some feature added at that version)